scripts/prepare-package-for-release.js

```diff
@@ -81,9 +81,20 @@ if (
   exit(1);
 }
 
+// Release builds should commit the version bumps, and create tags.
+echo('Updating RNTester react-native dependency...');
+if (
+  exec(`source scripts/update_rn_tester.sh && update_package_json ${version}`)
+    .code
+) {
+  echo('Failed to update RNTester react-native dependency.');
+  echo('Fix the issue, revert and try again.');
+  exit(1);
+}
+
 // Release builds should commit the version bumps, and create tags.
 echo('Updating RNTester Podfile.lock...');
-if (exec('source scripts/update_podfile_lock.sh && update_pods').code) {
+if (exec('source scripts/update_rn_tester.sh && update_pods').code) {
```

Comment:
`source scripts/update_rn_tester.sh` has already been sourced. We can skip it here:
```suggestion
if (exec('update_pods').code) {
```